packages/contracts/src/contracts/current/protocol/Exchange/MixinTransactions.sol

```diff
@@ -34,6 +37,30 @@ contract MixinTransactions is
     // Address of current transaction signer
     address public currentContextAddress;
 
+    bytes32 constant EXECUTE_TRANSACTION_SCHEMA_HASH = keccak256(
+        "ExecuteTransaction(",
```

Comment:
Should this be `executeTransaction`?

Reply:
I'd probably prefer just `Transaction`, but I could see how that might be confusing. Maybe `ZeroExTransaction`?